internal/datasource/host/exec/exec.go

```diff
@@ -18,6 +23,9 @@ type ExecInterface interface {
 	RunCmd(ctx context.Context, cmd string, args ...string) (*bytes.Buffer, error)
 	FindExecutable(name string) (string, error)
 	KillProcess(pid int32) error
+	GetHostname() (string, error)
+	GetHostID(ctx context.Context) (string, error)
+	GetReleaseInfo(ctx context.Context) (releaseInfo *v1.ReleaseInfo)
```

Comment:
should we add os.Getwd and os.Executable to here from config.go?

Reply:
Also, let's get rid of the `Get` prefix: https://go.dev/doc/effective_go#Getters